flixel/addons/display/FlxNestedSprite.hx

```diff
@@ -266,14 +266,18 @@ class FlxNestedSprite extends FlxSprite
 					var cos:Float = Math.cos(radians);
 					var sin:Float = Math.sin(radians);
 					
-					var dx:Float = child.relativeX - offset.x;
-					var dy:Float = child.relativeY - offset.y;
+					/* Fix from yannicular */	
```

Comment:
What's the point of this comment here? If you're concerned about credit - the commit history pretty much covers that.

We can't really put a comment in the source for the author of every fix, if we did readability would suffer. Additionally, only doing it in some cases would be unfair to other contributors.